.github/workflows/k8s_matrix_test.yaml

```
@@ -0,0 +1,14 @@
+# Runs the single-cluster e2e tests with various versions of Kubernetes, to ensure backward compatibility.
+name: K8s version test
+on:
+  schedule:
+    - cron: '0 12 * * 0' # Every Sunday at noon UTC
```

Adding workflow_dispatch allows to trigger the matrix manually, which can be convenient if we want to run the tests outside of the cron schedule.

```suggestion
  schedule:
    - cron: '0 12 * * 0' # Every Sunday at noon UTC
  workflow_dispatch:
```